tools/astarte_export/Dockerfile

```diff
@@ -37,7 +42,7 @@ RUN apt-get -qq install libssl1.1
 
 # We have to redefine this here since it goes out of scope for each build stage
 ARG BUILD_ENV=prod
-
+COPY ./run_export.sh /app/
 COPY --from=builder /app/_build/$BUILD_ENV/rel/astarte_export .
 
-CMD ["sh", "-c", "./bin/astarte_export export ${REALM} ${OUTPUT_XML_FILE}"]
+CMD /app/run_export.sh
```

Comment:
The preferred form for `CMD` is the so-called "exec" form, see https://docs.docker.com/engine/reference/builder/#cmd. Let's keep using best practices